wordpress-mobile/WordPress-iOS
# Issue/8020 xmlrpc api background uploads

Fixes #8020 

Updates the XMLRPC Api to be able to use background uploads. In order to be able to do background uploads the class was refactored to use Alamofire.

To test:
 - Start the app
 - Connect to a XMLRPC site
 - Test the several parts of the app, Post, Media, Comments, Site Settings and see if works correctly
 - Test the same with an XMLRPC site with basic auth.
